src/index.js

```diff
@@ -50,6 +36,7 @@ class ReactRefreshPlugin {
 
     // Inject refresh utilities to Webpack's global scope
     const providePlugin = new webpack.ProvidePlugin({
+      [errorOverlay]: this.options.overlay && require.resolve(this.options.overlay.module),
```

Comment:
There's an issue with line. Basically, this is resolving to `[errorOverlay]: false` so webpack is complaining that there's no module named false

![image](https://user-images.githubusercontent.com/1436588/77896883-368c6000-7279-11ea-8f22-6c455e1cf290.png)

If you remove the line, everything works fine (but of course you won't have the overlay)

Reply:
Yes, I noticed this. A fix is on its way.